raisdev/LibreChat
# SSL + Domain setup

## Description

* Updated `nginx.conf` with `chat.resarch.cornell.edu` URL and updated the rest of the file to get it to work with the AWS ALB vs managing the Nginx config/ssl certs inside the app. 
* Added PR template
* Removed `version:` in docker-compose files (no longer needed and is obsolete). 

## Type of Change

- *Maintenance* (non-breaking change which resolves a CVE, deprecation, or infrastructure issue/change)


## To Test

* Not really a way to test it locally other than to deploy it to check for the SSL config. 

## Notes

Please add any additional information below (if needed):

## Link to Asana task(s):
